css/index.css

```
@@ -1380,3 +1377,315 @@ body {
   }
 }
 
+
+.collections {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin: auto;
+  gap: 20px;
+  margin-top: 20px;
+  margin-bottom: 20px;
+}
+
+.collections__floors {
+  width: 90%;
+  height: auto;
+  border-radius: 12px;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.2);
+  background: linear-gradient(180deg, #0CEACF 0%, #0BA18F 100%);
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: row;
+  text-align: center;
+  position: relative;
+  color: #FFFFFF;
+  margin: auto;
+}
+
+.collections__link {
+  font-size: 0;
+  inset: 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.collections__svg {
+  width: 326.5px;
+  height: 398.75px;
+  display: flex;
+  position: absolute;
+}
+
+.collections__information {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  margin-top: 30px;
+  width: 50%;
+}
+
+.collections__title {
+  font-size: 14px;
+  font-weight: 600;
+  line-height: 20px;
+  letter-spacing: 1%;
+  color: #FFFFFF;
+  margin-right: auto;
+  margin-left: 22px;
+}
+
+.collections__text {
+  font-size: 24px;
+  font-weight: 700;
+  line-height: 28px;
+  letter-spacing: 1%;
+  text-align: left;
+  margin-left: 20px;
+  margin-top: 5px;
+}
+
+.collections__img {
+  width: 50%;
+  height: auto;
+  border-radius: 0 12px 12px 0;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.collections__botton {
+  display: flex;
+  justify-content: center;
+  margin-top: 32px;
+  background-color: #FFFFFF4D;
+  color: white;
+  font-size: 10px;
+  border-radius: 8px;
+  width:80%;
+  height: 40px;
+  text-decoration: none;
+  font-weight: 600;
+  line-height: 15.6px;
+  letter-spacing: 6%;
+  align-items: center;
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.07);
+}
+
+.collections__floors:hover {
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.6);
+}
+
+.collections__link:hover ~ .collections__information .collections__botton {
+  background-color: #3432324d;
+}
+
+.collections__painting {
+  box-shadow: 0 4px 8px rgba(0, 0, 0, 0.15);
+  background: linear-gradient(180deg, #F7CB20 0%, #F8A904 55.73%);
+  border-radius: 12px;
+  display: flex;
+  text-align: center;
+  width: 90%;
+  position: relative;
+  color: #FFFFFF;
+  margin: auto;
+  flex-direction: row;
+  height: 221.12px;
+}
+
+.collections__svg-right {
+  height: 221.12px;
+  opacity: 0px;
```

Esta propiedad no usa px como unidad, sino un valor entre cero y uno o un porcentaje.
https://developer.mozilla.org/en-US/docs/Web/CSS/opacity#values
```suggestion
  opacity: 0;
```